go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/gin-contrib/cors v1.3.1
 	github.com/gin-gonic/gin v1.7.4
 	github.com/golang-jwt/jwt/v4 v4.0.0
+	github.com/golang/protobuf v1.5.2
```

Comment:
not sure how we managed not to have this already?

Reply:
me either; that is a bit weird; before our next deploy we may want to run `go generate ./…` to make sure everything is up to date.